api/src/worker-start.ts

```diff
@@ -0,0 +1,4 @@
+const JobQueue = require("./services/JobQueue").default; // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
Does this line need to be cleaned up to make eslint happy, or is it really necessary to do it this way?

Reply:
Made change